### PR TITLE
CI: workflow file for testing cv_workflow

### PR DESCRIPTION
**Pull request by sugetha24**
## Change Summary

Workflow for running molecule test for `cv_workflow` plugin.

## Component(s) name

`arista.avd.cv_workflow`

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [ ] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [ ] My change requires a change to the documentation and documentation have been updated accordingly.
- [ ] I have updated [molecule CI](https://github.com/aristanetworks/avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

**Comment by sugetha24** [comment]
NOTE: This can be merged only after merging #3500

File: .github/workflows/new-cvp-integration.yml
Comment (ClausHolbechArista) [inline .github/workflows/new-cvp-integration.yml]: Please remove this if not needed.

File: .github/workflows/new-cvp-integration.yml
Comment (ClausHolbechArista) [inline .github/workflows/new-cvp-integration.yml]: We should not need this in a dispatched workflow.

File: .github/workflows/new-cvp-integration.yml
Comment (ClausHolbechArista) [inline .github/workflows/new-cvp-integration.yml]:
```suggestion
          - 'cv_workflow'
```

File: .github/workflows/new-cvp-integration.yml
Comment (ClausHolbechArista) [inline .github/workflows/new-cvp-integration.yml:31]: This secret is being printed many times throughout the run, so we cannot merge this until that has been addressed.

**Review by ClausHolbechArista — CHANGES_REQUESTED** [review]

File: .github/workflows/new-cvp-integration.yml
Comment (ClausHolbechArista) [inline .github/workflows/new-cvp-integration.yml]: update name here and in the heading above

File: .github/workflows/new-cvp-integration.yml
Comment (sugetha24) [reply .github/workflows/new-cvp-integration.yml:31]: Fixed with `no_log` in molecule test

**Review by ClausHolbechArista — APPROVED** [review]
LGTM. Thanks

File: .github/workflows/new-cvp-integration.yml
Comment (ClausHolbechArista) [inline .github/workflows/new-cvp-integration.yml]: We should cancel in-progress on this one, as it could leave the cloudvision server in some weird state.

File: .github/workflows/new-cvp-integration.yml
Comment (gmuloc) [inline .github/workflows/new-cvp-integration.yml]: There should be a newer version out
```suggestion
      - uses: actions/checkout@v4
```

File: .github/workflows/new-cvp-integration.yml
Comment (gmuloc) [inline .github/workflows/new-cvp-integration.yml]:
```suggestion
      - uses: actions/checkout@v4
```

**Review by gmuloc — APPROVED** [review]

**Pull request merged**
